packages/react-server-cli/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "author": "Sasha Aickin",
   "bin": {
-    "react-server-cli": "./bin/react-server-cli"
+    "react-server": "./bin/react-server-cli"
```

Comment:
`react-server startServer` is kinda cumbersome, though easily discoverable.  Iirc, you can have as many bin aliases as you want -- would it make sense to have an `rs` alias by default, or should we leave users to create their own aliases in `.bashrc` or `.zshrc`?